sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -203,5 +210,5 @@ case class UnresolvedGetField(child: Expression, fieldName: String) extends Unar
   override def eval(input: Row = null): EvaluatedType =
     throw new TreeNodeException(this, s"No function to evaluate expression. type: ${this.nodeName}")
 
-  override def toString: String = s"$child.$fieldName"
+  override def toString: String = s"$child.getField($fieldExpr)"
```

Comment:
Previously we use `$child[$fieldExpr]` to represent `GetItem`(for Map and Array) and use `$child.$fieldName` to represent `GetField`(for Struct), now they are combined, I have no idea how to name it...